my-app/src/components/cart.js

```diff
@@ -0,0 +1,134 @@
+import { useProducts } from "../contexts/ProductContext";
+import { useEffect } from "react";
+
+import "./cart.css";
+import { FaPlus, FaMinus } from "react-icons/fa";
+import { BsPlusSquare, BsDashSquare } from "react-icons/bs";
+import axios from "axios";
+import {
+   decrementQuantity,
+   deleteRequestCart,
+   incrementQuantity,
+   itemPrice,
+   moveToWishlist,
+   priceProductTotal,
+} from "./utilities/utilities";
+export function Cart() {
+   const { state, dispatch } = useProducts();
+
+   useEffect(() => {
+      (async function () {
+         try {
+            const productData = await axios.get(
+               "https://myfeline-restapi.sohamparab13.repl.co/cart"
+            );
+            console.log(productData);
+
+            dispatch({
+               type: "SET CART",
+               payload: productData.data,
+            });
+         } catch (error) {
+            dispatch({ type: "LOADING", payload: false });
+         }
+      })();
+   }, []);
+
+   return (
+      <div className="cart-parent">
+         <div className="card-holder">
+            {state &&
+               state.cart.map((item) => {
+                  return (
+                     <div>
+                        <div className="horizCardParent">
+                           <div className="horizCardBody">
+                              <div className="HorizImgDiv">
+                                 <img src={item.images} />
+                              </div>
+                              <div className="HorizCardDetails">
+                                 <div className="brandTitle">{item.name}</div>
+                                 <p className="prdDescrip">
+                                    {item.description}
+                                 </p>
+                                 <div className="horizCardBtnDiv">
+                                    <BsPlusSquare
+                                       // onClick={() => {
```

Comment:
code comments should be removed. Accidental uncomment can cause bugs or errors in the site.